zappa/zappa.py

```diff
@@ -736,16 +736,30 @@ def rollback_lambda_function_version(self, function_name, versions_back=1, publi
 
         return response['FunctionArn']
 
+    def get_lambda_function(self, function_name):
+        """
+        Returns the lambda function ARN, given a name
+
+        This requires the "lambda:GetFunction" role.
+        """
+        response = self.lambda_client.get_function(
+                FunctionName=function_name)
+        return response['Configuration']['FunctionArn']
+
     def get_lambda_function_versions(self, function_name):
         """
         Simply returns the versions available for a Lambda function, given a function name.
 
         """
         try:
-            response = self.lambda_client.list_versions_by_function(FunctionName=function_name)
+            response = self.lambda_client.list_versions_by_function(
+                FunctionName=function_name
+            )
             return response.get('Versions', [])
         except Exception:
-            return []
+            # reference: https://github.com/Miserlou/Zappa/issues/617
+            print('You might need "lambda:ListVersionsByFunction" policy enabled')
```

Comment:
I don't agree with this change.

Not everybody needs this policy, it's far more likely that you're just trying to call status on a function that doesn't exist rather than one you don't have the ability to access. If we're going to state that it's for permission reasons, we should do it when it is actually for permission reasons.